opm/material/fluidmatrixinteractions/EclTwoPhaseMaterial.hpp

```diff
@@ -163,6 +163,72 @@ class EclTwoPhaseMaterial : public TraitsT
         }
     }
 
+    /*
+     * Hysteresis parameters for oil-water
+     * @see EclHysteresisTwoPhaseLawParams::pcSwMdc(...)
+     * @see EclHysteresisTwoPhaseLawParams::krnSwMdc(...)
+     * \param params Parameters
+     */
+    template <class Evaluation>
+    static void oilWaterHysteresisParams(Evaluation& pcSwMdc,
+                                         Evaluation& krnSwMdc,
+                                         const Params& params)
+    {
+        pcSwMdc = params.oilWaterParams().pcSwMdc();
+        krnSwMdc = params.oilWaterParams().krnSwMdc();
+
+        Valgrind::CheckDefined(pcSwMdc);
+        Valgrind::CheckDefined(krnSwMdc);
+    }
+
+    /*
+     * Hysteresis parameters for oil-water
+     * @see EclHysteresisTwoPhaseLawParams::pcSwMdc(...)
+     * @see EclHysteresisTwoPhaseLawParams::krnSwMdc(...)
+     * \param params Parameters
+     */
+    template <class Evaluation>
+    static void setOilWaterHysteresisParams(const Evaluation& pcSwMdc,
+                                            const Evaluation& krnSwMdc,
+                                            Params& params)
+    {
+        const double krwSw = 2.0; //Should not be used
```

Comment:
use `Scalar` instead of `double`. also this function will break if `Evaluation != double`. (but Scalars instead of Evaluations are sufficient as arguments for this method anyway as far as I can see.)